src/EditorFeatures/CSharp/AutomaticCompletion/Sessions/CurlyBraceCompletionSession.cs

```diff
@@ -108,7 +110,8 @@ private static bool ContainsOnlyWhitespace(IBraceCompletionSession session)
 
         private IEnumerable<AbstractFormattingRule> GetFormattingRules(Document document)
         {
-            return SpecializedCollections.SingletonEnumerable(BraceCompletionFormattingRule.Instance).Concat(Formatter.GetDefaultFormattingRules(document));
+            var indentStyle = document.GetOptionsAsync(CancellationToken.None).WaitAndGetResult_CanCallOnBackground(CancellationToken.None).GetOption(FormattingOptions.SmartIndent);
```

Comment:
Do we need a synchronous `GetOptions` or `GetOption` helper here or you think that will likely be misused?

Reply:
I'm hoping to see `document.GetOptionsAsync` reduce in usage over time so most of these cases will naturally go away.